### ISSUE TITLE
avoid duplicating clinical trials by checking if the eduract or nct number already exists 

### ISSUE THREAD

**Issue by brunoamaral** [brunoamaral/gregory#314]

**Comment by brunoamaral** [comment]
```
from django.db.models import Q

nct = "NCT12345" # replace with your nct value
if ClinicalTrial.objects.filter(Q(identifiers__nct=nct)).exists():
    # nct already exists in the database
    pass
else:
    # nct is not in the database
    pass

```

**Comment by brunoamaral** [comment]
fixed in 783780a375ccc6066894d155bd94853f6014ba57
